.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["@babel/plugin-transform-modules-commonjs"]
```

Comment:
this only targets jest right now and allows watch mode to operate on source files, transpiling on the fly